chapters/10-community-involvement.Rmd

```diff
@@ -9,7 +9,28 @@
 - NEWS file
 - reprex
 
+## Outline of action steps
 
+This chapter is very description-heavy with minimal code.
+
+- Creating a welcoming community with welcoming materials
+<!-- I grouped reprex and NEWS with the README because they seem related to the general welcoming presentation of a project, but they could go elsewhere. -->
+  - Text: the purpose of a README and what goes into one ([Mozilla reference](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), [R packages reference](https://r-pkgs.org/release.html#readme)).
```

Comment:
@k8hertweck, how about this more specific language around modifying a README to be welcoming? 

```suggestion
  - Text: making your README welcoming and easy to understand ([Mozilla reference](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), [R packages reference](https://r-pkgs.org/release.html#readme)).
```